node/completion.bash

```diff
@@ -0,0 +1,4 @@
+export NVM_DIR=/usr/local/var/nvm
+
+nvm_completion="$(brew --prefix nvm)/nvm.sh"
```

Comment:
@maratfakhreev looks like `nvm_completion` is incorrect name for this file b/c this file contains `nvm` actually.
And bash completion should be loaded in different way https://github.com/creationix/nvm#bash-completion